### ISSUE TITLE
Display trash icons by default

### ISSUE THREAD

**Issue by marlenamellody** [hackforla/tdm-calculator#2383]
### Overview
Make trash can icons appear by default on the Admin FAQ and Admin Guide pages, not just upon hover.

### Details
Admins can edit content within the FAQ and Admin Guide pages. Currently, the trash icon only appears on hover, but it should be visible by default.

<details><summary>Trash icon not visible</summary>

<img width="1038" alt="Image" src="https://github.com/user-attachments/assets/92ab39dd-02ff-4a6a-a0af-20a5ce650c8a" />
<img width="1040" alt="Image" src="https://github.com/user-attachments/assets/28ca3c58-6776-4e09-98ed-a353b8826af9" />
</details>

<details><summary>Trash icon visible upon hover</summary>

<img width="905" alt="Image" src="https://github.com/user-attachments/assets/9bb7a705-ae8a-476b-ab7c-5ae46c0c73d5" />

<img width="862" alt="Image" src="https://github.com/user-attachments/assets/17656bb2-8b34-49c5-8e48-4c254ee389e5" />

</details>

Note: The Admin Guide has not been implemented yet, but it will eventually reflect the Admin FAQ page.

### Action Items
- [x] When working on the FAQ page, do not edit existing questions and answers - they have been carefully created and refined by the stakeholders. Instead, you can create new categories and/or FAQs to experiment with, but be sure to delete the experimental data when you are done.
- [x] Assign yourself to the development issue.
- [x] Move the development issue from the Prioritized Backlog to In Progress.
- [x] [ADD DEVELOPER ACTION ITEMS]
- [x] Create a PR, if applicable.
- [ ] Once the PR has been approved, use it to close this issue. If there is no PR associated with this issue, close this issue.

### Resources/Instructions

Steps to navigate to Edit FAQ page: 

1. [Sign in](https://tdm-dev.azurewebsites.net/login) as Admin
2. Navigate to [FAQ](https://tdm-dev.azurewebsites.net/faqs)
3. Click 'Edit FAQ page' button

- [Added to staging deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit#slide=id.g34d039f4ac3_10_0)

**Comment by marlenamellody** [comment]
Please add missing labels.
